pypeit/spectrographs/keck_mosfire.py

```diff
@@ -269,6 +270,10 @@ def init_meta(self):
         self.meta['idname'] = dict(card=None, compound=True)
         self.meta['frameno'] = dict(ext=0, card='FRAMENUM')
         self.meta['object'] = dict(ext=0, card='OBJECT')
+        self.meta['decker_basename'] = dict(card=None, compound=True)
```

Comment:
It seems to me that you could be using decker instead of decker_basename here and then we don't need this additional meta variable.

Reply:
See my response to your previous comment about `decker` and `decker_basename`.